IFC4x3/Templates/Project Context/Project Global Positioning Geodetic/DocTemplateDefinition.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocTemplateDefinition xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Project_Global_Positioning_Geodetic_3rJ3aKTBj11wk7gKUHSOa" Name="Project Global Positioning Geodetic" UniqueId="f54c3914-74bb-4107-ab87-fea51e45c624" Type="IfcContext">
+	<Rules>
+		<DocModelRuleAttribute Name="RepresentationContexts">
+			<Rules>
+				<DocModelRuleEntity Name="IfcGeometricRepresentationContext">
+					<Rules>
+						<DocModelRuleAttribute Name="HasCoordinateOperation" Identification="HasGlobalPosition">
+							<Rules>
+								<DocModelRuleEntity Name="IfcRigidOperation" Identification="FirstCoordinate">
```

Comment:
How come the RuleID is `FirstCoordinate`? This should have been on an attribute, I suppose.

Reply:
See [2e01595](https://github.com/bSI-InfraRoom/IFC-Specification/pull/506/commits/2e01595d7b5045cab93d09be9f9ccb670ebc266c)